WordPress/Classes/Models/Comment.m

```diff
@@ -102,8 +102,8 @@ - (NSDate *)dateCreated
 
 - (BOOL)isPrivateContent
 {
-    if ([self.post respondsToSelector:@selector(isPrivate)]) {
-        return (BOOL)[self.post performSelector:@selector(isPrivate)];
+    if ([self.post respondsToSelector:@selector(isPrivateAtWPCom)]) {
+        return (BOOL)[self.post performSelector:@selector(isPrivateAtWPCom)];
```

Comment:
I've looked at BasePost, AbstractPost and ReaderPost, and ran `ack isPrivateAtWPCom`.  I'm not finding any where that post implements `isPrivateATWPCom`.   Was this supposed to be modified to check `self.post.blog` or, given the change to the AbstractPost's `isPrivate` method, should this change be rolled back?

Reply:
I think this was a mistake in the auto-renaming tool in Xcode.  Nice catch!

Reply:
Silly Xcode :)

Reply:
I rolled back this change as a result of the rename below.